tests/conftest.py

```diff
@@ -1,3 +1,4 @@
+import re
```

Comment:
It was being used when I was using a regex instead of string comparisons but it's not anymore.